### PR TITLE
Fixed one test that was causing error

### PR DESCRIPTION
**Pull request by aanm**
Fixed one test that was causing error

Signed-off-by: André Martins martins@noironetworks.com

### CODE REVIEW

**Comment by crosbymichael** [comment]
LGTM

File: integration-cli/docker_api_containers_test.go [hunk 1]
```diff
@@ -293,7 +293,7 @@ func TestGetContainerStats(t *testing.T) {
 		dec := json.NewDecoder(bytes.NewBuffer(sr.body))
 		var s *stats.Stats
 		// decode only one object from the stream
-		if err := dec.Decode(&s); err != nil {
+		if err := dec.Decode(&s); err != nil && err != io.EOF {
```
Comment (LK4D4) [inline integration-cli/docker_api_containers_test.go]: Hmm, could you explain pls? Because I think that getting `EOF` on unmarshal is definetely error.

Comment (aanm) [inline integration-cli/docker_api_containers_test.go]: @LK4D4 It is? I'm getting that error on Fedora. I thought it was "normal" to have a `EOF` error

Comment (jfrazelle) [inline integration-cli/docker_api_containers_test.go]: there seems to be a different issue here then, because afaik EOF is not a normal error

**Comment by tiborvass** [comment]
@aanm could you provide us with an environment where you see this error?

**Comment by LK4D4** [comment]
I saw that error too. It is a race in test, seems like we're trying to get stats before any was written

**Comment by tiborvass** [comment]
@LK4D4 ok so you agree that the `err != io.EOF` part should not be there, right? The rest of the PR is acceptable.

**Comment by jfrazelle** [comment]
ooooo noooo this is the systemd error sorry

**Comment by jfrazelle** [comment]
so can we just remove that part of this PR, we are aware of the problems with this test on systemd and would like the other fixes just not the `EOF` ones

**Comment by aanm** [comment]
@jfrazelle I've removed the EOF stuff. Is there an issue createad regarding that?

**Comment by tiborvass** [comment]
LGTM

**Pull request merged**